schema/spacewalk/common/tables/rhnErrata.sql

```diff
@@ -27,6 +27,9 @@ CREATE TABLE rhnErrata
 							    'Security Advisory')),
     advisory_name     VARCHAR(100) NOT NULL,
     advisory_rel      NUMERIC NOT NULL,
+    advisory_status   VARCHAR(32) NOT NULL DEFAULT('final')
+                          CONSTRAINT rhn_errata_adv_status_ck
+			     CHECK (advisory_status in ('final', 'testing', 'retracted')),
```

Comment:
is there a list somewhere of all the status values and what they mean? I have not seen `testing` before but i have seen `stable` as status value.

Reply:
Outcome: there is no fixed list. We fould also "stable" and use now these 4 values and see if customers find more.